veracruz-server/src/veracruz_server_nitro.rs

```diff
@@ -68,7 +68,6 @@ pub mod veracruz_server_nitro {
                 enclave: runtime_manager_enclave,
             };
             println!("VeracruzServerNitro::new Runtime Manager instantiated. Calling initialize");
-            std::thread::sleep(std::time::Duration::from_millis(10000));
```

Comment:
Can you keep this delay, albeit reduce it to 2-3 seconds?

Reply:
Sounds like the easiest and most sensible thing to do